packages/babel-cli/src/babel/index.js

```diff
@@ -221,6 +225,11 @@ if (commander.deleteDirOnStart && !commander.outDir) {
   errors.push("--delete-dir-on-start requires --out-dir");
 }
 
+if (commander.settings) {
+  printSettings();
+  process.exit(2);
```

Comment:
Why exit code 2, do we have a manual specifying this? Or conventional?

Reply:
it should be empty,  

> Or conventional?

yes, https://nodejs.org/api/process.html#process_exit_codes